web/client/codechecker_client/credential_manager.py

```diff
@@ -21,7 +21,8 @@
 from codechecker_common.logger import get_logger
 from codechecker_common.util import load_json_or_empty
 
-from codechecker_web.shared.env import check_file_owner_rw, get_password_file
+from codechecker_web.shared.env import check_file_owner_rw, get_password_file, \
```

Comment:
Failed on pycodestyle:
```
client/codechecker_client/credential_manager.py:24:80: E501 line too long (80 > 79 characters)
```

Reply:
Wasn't aware that the line length was 79, not 80. Thanks for spotting!

Fixed